### ISSUE TITLE
auth.server.cafile configuration doesn't take effect

### ISSUE THREAD

**Issue by xiefeifeihu** [hivemq/mqtt-cli#125]
## Expected behavior
When config auth.server.cafile in config.properties,it doesn't need to add --cafile in command line.

## Actual behavior
When config auth.server.cafile in config.properties,Command Line doesn't take effect.
It also need to add --cafile in command line.

## Details
- Affected MQTT CLI version(s): 1.1.1
- Used JVM version: 13.0.1 (Oracle Corporation Java HotSpot(TM) 64-Bit Server VM 13.0.1+9)
- OS: Windows 10 10.0 amd64

**Comment by mathias-ewald** [comment]
I can confirm.

```
$ mqtt --version
1.1.1
Picocli 4.0.4
JVM: 13 (Oracle Corporation OpenJDK 64-Bit Server VM 13+33)
OS: Mac OS X 10.14.6 x86_64
```

**Comment by mathias-ewald** [comment]
I might have found the problem in the code [here](https://github.com/hivemq/mqtt-cli/blob/develop/src/main/java/com/hivemq/cli/DefaultCLIProperties.java#L128)

It's filtering properties contained in `propertyToValue`but that does not contain `SERVER_CERTIFICATE`

**Comment by xiefeifeihu** [comment]
I have fixed the bug。Now I'm dealing with the pull request.

[](https://github.com/hivemq/mqtt-cli/pull/123)

Can you see that?

**Comment by gitseti** [comment]
resolved with #123
